extensions/amp-story/1.0/test/test-utils.js

```diff
@@ -151,4 +155,40 @@ describes.fakeWin('amp-story utils', {}, (env) => {
       );
     });
   });
+
+  describe('dependsOnStoryServices', () => {
```

Comment:
Mainly for safety, and to ensure that we're able to short-circuit in tests. See comment in `dependsOnStoryServices` implementation below.